src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+function Footer(props) {
+  return (
+    <Wrapper>
+      <Container to="/">
+        <Product src="./img/painted-bottle.jpg" alt="Products" />
+      </Container>
+      <Container to="/motive">
+        <Motiv src="./img/Logo.jpg" alt="Motive" />
+      </Container>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.footer`
+  display: grid;
+  grid-template-areas: 'product campaign cocktails';
+  justify-content: flex-start;
+  align-items: center;
+  margin: 0 auto;
+  margin-bottom: 10px;
+  border-radius: 19px;
+  background-color: white;
+  width: 22rem;
+  height: 5rem;
+`
+const Container = styled(Link)``
```

Comment:
You dont really need this one when you dont give it any styling. Just use the `Link` in the return

Reply:
You dont have to do it in order for it to work properly. 
It would be more semantic:
[HTML5 Semantic Tags: What They Are and How to Use Them!
](https://www.semrush.com/blog/semantic-html5-guide/)
In the **Using sections and articles** area of the article you can see a picture that descibes it